src/internal/modules/billing/routes.js

```diff
@@ -103,6 +103,11 @@ if (isAcceptanceTestTarget) {
             pageTitle: 'Bill runs',
             activeNavLink: 'notifications'
           }
+        },
+        validate: {
+          query: {
+            page: Joi.number().default(1)
```

Comment:
I wonder if this should be:
Joi.number().integer().min(1).default(1)

Reply:
done